src/EscrowERC20.sol

```diff
@@ -53,6 +54,18 @@ contract EscrowERC20 {
         playerToContractToAmount[playerAddress][contractAddress] = _Amount;
     }
 
+    function transferTokens(address _PlayerAddress, address _ContractAddress, uint256 _Amount)
+        public
```

Comment:
This function should be external if it's only going to be called from outside, i.e if no function inside the Escrow contract is calling this function, then the function should be `external`